index.html

```diff
@@ -239,6 +239,30 @@ <h2>LINMOB</h2>
 
         <hr />
 
+        <div class="row">
+
+          <!-- User interface project card -->
+          <div class="col-md-4" id="project9">
+            <h2>Phosh</h2>
+            <div class="fullwidth">
+              <img class="card-img-top oslogo" src="images/phosh.mobi.svg" alt="Phosh logo: A tiger holding a sign reading 'Phosh'">
```

Comment:
The `images/phosh.mobi.svg` image seems to be missing from the PR. If you could add it in please that would be great!